### PR TITLE
Users/kcheekuri/remove phoenix flag

### PR DESCRIPTION
**Pull request by kcheekuri**
[Preview this branch](https://ms.portal.azure.com/?feature.phoenixendpoint=https://juno-test.documents-dev.windows-int.net&dataExplorerSource=https://cosmos-explorer-preview.azurewebsites.net/commit/a7409553d40151f18c5ab4fc999fa3be76706357/explorer.html?feature.pr=https://github.com/Azure/cosmos-explorer/pull/1160#home)

This PR:

- Removes phoenix feature flag and notebooks temporary down feature flag
- Allow only whitelisted accounts for phoenix
- Updated api end points for allocate, reset and container health check

### CODE REVIEW

File: src/Phoenix/PhoenixClient.ts [hunk 1]
```diff
@@ -117,6 +134,24 @@ export class PhoenixClient {
...
+        }
+      );
+      return response.status === HttpStatusCodes.OK;
+    } catch (error) {
+      Logger.logError(getErrorMessage(error), "PhoenixClient/IsDbAcountWhitelisted");
+      return false;
+    }
+  }
+
   public static getPhoenixEndpoint(): string {
```
Comment (srinathnarayanan) [inline src/Phoenix/PhoenixClient.ts:155]: new endpoint is of the form api/controlplane/toolscontainer/cosmosaccounts

File: src/Common/Constants.ts [hunk 2]
```diff
@@ -96,7 +96,6 @@ export class Flights {
   public static readonly AutoscaleTest = "autoscaletest";
   public static readonly PartitionKeyTest = "partitionkeytest";
   public static readonly PKPartitionKeyTest = "pkpartitionkeytest";
-  public static readonly Phoenix = "phoenix";
```
Comment (swviswan) [inline src/Common/Constants.ts:99]: >Phoenix [](http://example.com/codeflow?start=25&length=7)

why do you want to remove this? let me sync with you offline

Comment (kcheekuri) [reply src/Common/Constants.ts:99]: ok

Comment (swviswan) [reply src/Common/Constants.ts:99]: Please dont remove the flight. We will be flighting when we roll out in prod.

**Pull request closed without merging**